src/game_api/script/usertypes/vtables_lua.cpp

```diff
@@ -18,58 +26,109 @@ void register_usertypes(sol::state& lua)
+        // <"get_animation_map", 41, MemFun<&Movable::get_animation_map>>,
+        VTableEntry<"apply_velocity", 42, MemFun<&Movable::apply_velocity>>,
+        /// NoDoc
+        VTableEntry<"stomp_damage", 43, MemFun<&Movable::get_damage>>,
+        VTableEntry<"get_damage", 43, MemFun<&Movable::get_damage>>,
+        // <"get_stomp_damage", 43, MemFun<&Movable::get_stomp_damage>>, // still don't know the difference between this and get_damage
+        VTableEntry<"is_on_fire", 45, MemFun<&Movable::is_on_fire>>,
+        VTableEntry<"attack", 46, MemFun<&Movable::attack>>,
+        VTableEntry<"thrown_into", 47, MemFun<&Movable::thrown_into>>,
+        VTableEntry<"damage", 48, MemFun<&Movable::damage>>,
+        VTableEntry<"on_hit", 49, MemFun<&Movable::on_hit>>,
+        VTableEntry<"get_damage_sound", 50, MemFun<&Movable::get_damage_sound>>,
+        VTableEntry<"stun", 51, MemFun<&Movable::stun>>,
+        VTableEntry<"freeze", 52, MemFun<&Movable::freeze>>,
+        VTableEntry<"light_on_fire", 53, MemFun<&Movable::light_on_fire>>,
+        VTableEntry<"set_cursed", 54, MemFun<&Movable::set_cursed>>,
+        VTableEntry<"web_collision", 55, MemFun<&Movable::on_spiderweb_collision>>,
+        // set_last_owner_uid
+        // get_last_owner_uid
+        VTableEntry<"check_out_of_bounds", 58, MemFun<&Movable::check_out_of_bounds>>,
+        VTableEntry<"set_standing_on", 59, MemFun<&Movable::set_standing_on>>,
+        VTableEntry<"standing_on", 60, MemFun<&Movable::standing_on>>,
+        VTableEntry<"stomped_by", 61, MemFun<&Movable::on_stomped_on_by>>,
+        VTableEntry<"thrown_by", 62, MemFun<&Movable::on_thrown_by>>,
+        VTableEntry<"cloned_to", 63, MemFun<&Movable::copy_extra_info>>,
+        // get_type_id
+        // doesnt_have_spikeshoes (potentially wrong name)
+        // is_player_mount_or_monster
+        VTableEntry<"pick_up", 67, MemFun<&Movable::pick_up>>,
+        /// NoDoc
+        VTableEntry<"picked_up_by", 68, MemFun<&Movable::can_be_picked_up_by>>,
+        VTableEntry<"can_be_picked_up_by", 68, MemFun<&Movable::can_be_picked_up_by>>,
+        VTableEntry<"drop", 69, MemFun<&Movable::drop>>,
+        VTableEntry<"collect_treasure", 70, MemFun<&Movable::collect_treasure>>,
+        VTableEntry<"apply_movement", 71, MemFun<&Movable::apply_movement>>,
+        // damage_entity
+        // v73
+        VTableEntry<"is_powerup_capable", 74, MemFun<&Movable::is_powerup_capable>>, // dunno if called by the game, might be inlined?
+        VTableEntry<"initialize", 75, MemFun<&Movable::initialize>>,
+        // check_is_falling
+        // v77
+        VTableEntry<"process_input", 78, MemFun<&Movable::process_input>>,
+        // post_collision_damage_related?
+        VTableEntry<"picked_up", 80, MemFun<&Movable::on_picked_up>>,
+        VTableEntry<"release", 81, MemFun<&Movable::on_release>>,
+        VTableEntry<"generate_fall_poof_particles", 82, MemFun<&Movable::generate_fall_poof_particles>>,
```

Comment:
so maybe get rid of the "poof" part? or you have better name suggestion?

Reply:
`generate_fall_fx` or `generate_fall_effects`? Maybe add `floor_` before `fall`

Reply:
`generate_landing_effects`, maybe `on_landing` if it's not only effects perhaps?

Reply:
`generate_floor_landing_effects` ? fell like the floor is important here, since it does not work on stomping enemies, which can also be considered "landing on something". Or maybe just explain it in the comment since it's kind of long?

I still need to add comment that it does not generate the damage sounds or floor sounds when you fall while being stunned etc. (thrown by something for example)

Reply:
Either the long name or comment would work. Landing alone may not be clear enough if stomping counts, but it's probably enough to make someone doubt if it does and check the comment